doc/env_vars.md

```diff
@@ -141,5 +141,12 @@ _NOTE:_ `LRSQL_STMT_RETRY_LIMIT` and `LRSQL_STMT_RETRY_BUDGET` are used to mitig
 | `LRSQL_OIDC_CLIENT_TEMPLATE`      | `oidcClientTemplate`     | An optional template to modify LRS Admin UI client OIDC configuration. No effect if `LRSQL_OIDC_ISSUER` is not set.                                                                                                                                    | Not Set                                                                                    |
 | `LRSQL_OIDC_VERIFY_REMOTE_ISSUER` | `oidcVerifyRemoteIssuer` | Verify on startup that the issuer in remote configuration matches `LRSQL_OIDC_ISSUER`. No effect if `LRSQL_OIDC_ISSUER` is not set.                                                                                                                    | `true`                                                                                     |
 | `LRSQL_OIDC_ENABLE_LOCAL_ADMIN`   | `oidcEnableLocalAdmin`   | Whether or not to enable local administrative account authentication, login and management when `LRSQL_OIDC_ISSUER` is set. No effect if `LRSQL_OIDC_ISSUER` is not set.                                                                               | `false`                                                                                    |
+| `LRSQL_SEC_HEAD_HSTS`   | `secHeadHsts`   | Sets HTTP security response header `Strict-Transport-Security`. When not set, the header is omitted. When set to any string other than `"[default]"` it will use that value as the header. When set to `"[default]"` it will have the framework default (which is at time of writing this: `max-age=31536000; includeSubdomains`)                                                                                | Not Set                                                                                    |
```

Comment:
Ok I think it's time that we start splitting up the ginormous table for webserver config in the README (the config files themselves can stay the same). That way we can also add custom columns for certain categories like the security header vars.

Reply:
okay, propose some subcategories and i'll take care of it.

Reply:
I agree btw, while writing it i was like dang this table is nearly unreadable, AND in practice sometimes when im answering customers the closest anchor link i can give them i `webserver` and that is a LOT of scrolling for em haha

Reply:
Ok since you didn't have a proposal here's mine for webserver subsections:

# TLS & Ports

LRSQL_KEY_FILE
LRSQL_KEY_ALIAS
LRSQL_KEY_PASSWORD
LRSQL_KEY_PKEY_FILE
LRSQL_KEY_CERT_CHAIN
LRSQL_KEY_ENABLE_SELFIE
LRSQL_ENABLE_HTTP
LRSQL_ENABLE_HTTP2
LRSQL_HTTP_HOST
LRSQL_HTTP_PORT
LRSQL_SSL_PORT


# JWT Config

LRSQL_JWT_EXP_TIME
LRSQL_JWT_EXP_LEEWAY
LRSQL_JWT_NO_VAL
LRSQL_JWT_NO_VAL_UNAME
LRSQL_JWT_NO_VAL_ISSUER
LRSQL_JWT_NO_VAL_ROLE_KEY
LRSQL_JWT_NO_VAL_ROLE

# OIDC

LRSQL_OIDC_ISSUER
LRSQL_OIDC_AUDIENCE
LRSQL_OIDC_CLIENT_ID
LRSQL_OIDC_CLIENT_TEMPLATE
LRSQL_OIDC_VERIFY_REMOTE_ISSUER
LRSQL_OIDC_ENABLE_LOCAL_ADMIN

# CORS

LRSQL_ALLOW_ALL_ORIGINS
LRSQL_ALLOWED_ORIGINS

# Browser Security Headers

LRSQL_SEC_HEAD_HSTS
LRSQL_SEC_HEAD_FRAME
LRSQL_SEC_HEAD_CONTENT_TYPE
LRSQL_SEC_HEAD_XSS
LRSQL_SEC_HEAD_DOWNLOAD
LRSQL_SEC_HEAD_CROSS_DOMAIN
LRSQL_SEC_HEAD_CONTENT

# Paths & Features

LRSQL_URL_PREFIX
LRSQL_ENABLE_ADMIN_UI
LRSQL_ENABLE_ADMIN_STATUS
LRSQL_ENABLE_STMT_HTML

Reply:
@milt i'd like your thoughts on this as well.

Reply:
Those sound fine to me. Will this also be the case in code or just in docs?

Reply:
Sorry I didn't see this until now. I think for the most part it's fine, though I suggest that:
- "TLS & Ports" should be divided into "Host & Ports" and "TLS/SSL", in that order (since host + ports are arguably the most important config vars).
- "Pats & Features" sounds rather vague. I wish we could title this "UI" or "Admin," but those titles would only accurately describe 2 of the 4 vars each. Maybe just "Miscellaneous"?

Reply:
And @milt this is just in the docs, for now at least.

Reply:
yeah im ok with that

Reply:
https://github.com/yetanalytics/lrsql/pull/329/commits/c8694404bf2e87c137966271dd748a54d9da23ac

Reply:
Not sure about putting the HTTP and HTTP/2 vars under "Hosts & Ports" since they're not literally host and ports; maybe rename this section to "Basic Config." Also it may be helpful to add blurbs under "JWT Config" and "OIDC" to cut down on repeated sentences (e.g. "No effect if `LRSQL_OIDC_ISSUER` is not set." that is repeated for the OIDC vars). Otherwise looks good.

Reply:
@kelvinqian00 Agree on OIDC, easy aggregation there (175011173fc3f314ddadcc1b8c0844cd8cab4cea). JWT is not so cleanly or efficiently aggregated. 

Your first point with renaming has crossed over into nitpicky, and also isn't even totally true it does affect ports very much. Also don't ask to rename something in a review comment and when someone changes to _your_ name for it, change your mind in the review, especially when it's of less than zero consequence.

Reply:
Ok